### PR TITLE
Fix for #89

### LINKED ISSUE #89: Html style inside loop breaks the template
In a huge object loop (a project bean), I need to display in the report an html field value for wich project.

I read the documentation and successfully display a simple html field outside loop. But is it not possible to make it work in loop. How to do it?

You can checkout a simple template example that fail here https://github.com/kevinsdooapp/xdocreport-issue-77.git (I reused a now useless repository ;))

### LINKED ISSUE #93: Html fields could not be outside a paragraph
I'm writing a test case that you'll find in attachment of my pull request for the issue 89.

### PR DESCRIPTION
**Pull request by gumara-dev**
Hello,
thanks for your great work on this project. I would like to give something back and have fixed the following issue.
Resolves #89

### CODE REVIEW

File: template/fr.opensagres.xdocreport.template/src/main/java/fr/opensagres/xdocreport/template/formatter/IDocumentFormatter.java [hunk 1]
```diff
@@ -90,4 +90,6 @@ String formatAsCallTextStyling( long variableIndex, String fieldName, String doc
     String getEndNoParse();
 
     String getDefineDirective( String name, String value );
+    
+    boolean isInstruction( String tagContent );
```
Comment (angelozerr) [inline template/fr.opensagres.xdocreport.template/src/main/java/fr/opensagres/xdocreport/template/formatter/IDocumentFormatter.java:94]: Please add javadoc

File: template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/VelocityDocumentFormatter.java [hunk 2]
```diff
@@ -469,15 +471,21 @@ public String formatAsCallTextStyling( long variableIndex, String fieldName, Str
                                            String entryName )
     {
         StringBuilder newContent = new StringBuilder( START_SET_DIRECTIVE );
+        newContent.append(VAR_SYNTAX_WITH_DIRECTIVE);
+        newContent.append( EQUALS );
+        newContent.append( syntaxWithDirective );
+        newContent.append( END_SET_DIRECTIVE );
```
Comment (gumara-dev) [inline template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/VelocityDocumentFormatter.java:477]: This line will be removed at the next commit

File: template/fr.opensagres.xdocreport.template/src/main/java/fr/opensagres/xdocreport/template/formatter/IDocumentFormatter.java (hunk 1 above)
Comment (gumara-dev) [reply template/fr.opensagres.xdocreport.template/src/main/java/fr/opensagres/xdocreport/template/formatter/IDocumentFormatter.java:100]: Done

**Comment by angelozerr** [comment]
Thanks @gumara-dev !

**Pull request merged**